.gitignore

```diff
@@ -5,3 +5,5 @@
 .idea
 /deployment/data/nginx/app.conf
 .vscode/launch.json
+*.sw?
+package-lock.json
```

Comment:
@Weves, is it standard to have the package-lock.json be ignored? Doesn't this enforce the version consistency for the frontend packages?

Reply:
Ah, sorry, didn't see that it was already checked in, will add in #1631.